### PR TITLE
Documentation update for Face BUILD release

### PR DESCRIPTION
**Pull request by huxuan**
- Add `HowtoAddFaces` in `Face-API-How-to-Topics`.

### CODE REVIEW

File: Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- [Step 3: Create the persons to the person group](#step3)
+- [Step 4: Add faces to the persons](#step4)
+- [Summary](#summary)
+
+## <a name="step1"></a> Step 1: Initialization
+
+Several variables are declared and a helper function is implemented to schedule the requests.
+
+- `PersonCount` is the toal number of persons.
+- `CallLimitPerSecond` is the maximum calls per second according to the subscription.
```
Comment (wangjun-microsoft) [inline Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md]: `CallLimitPerSecond` is the maximum calls per second according to the subscription tier.

File: Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+- `_timeStampQueue` is a Queue to record the request timestamps.
+- `await WaitCallLimitPerSecond()` will wait until it is valid to send next reqeust.
+
+```CSharp
+const int PersonCount = 10000;
+const int CallLimitPerSecond = 10;
+static Queue<DateTime> _timeStampQueue = new Queue<DateTime>(CallLimitPerSecond);
+
+static async Task WaitCallLimitPerSecond()
```
Comment (wangjun-microsoft) [inline Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md]: rename as "WaitCallLimitPerSecondAsync"?

**Review by wangjun-microsoft — APPROVED** [review]
approved with two minor comments.

File: Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md (hunk 1 above)
Comment (huxuan) [reply Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md]: Done.

File: Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md (hunk 2 above)
Comment (huxuan) [reply Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md]: Done.

**Pull request closed without merging**